### PR TITLE
New package: Apache.Tomcat9 version 9.0.89

### PR DESCRIPTION
**Pull request by jaov**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/153777)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-05-15 15:50:18.077 [CLI ] ShellExecute installer failed: 2

Automated error analysis suggests -1978335226 may mean Install successful but system restart may be required.

(Automated response - build 881.)

**Comment by stephengillie** [comment]
Manual Validation ended with the same failure code `2`. Running the installer manually suggests that `Oracle.JavaRuntimeEnvironment` or another `JRE` might be a dependency: 
image

File: manifests/a/Apache/Tomcat9/9.0.89/Apache.Tomcat9.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Apache.Tomcat9
+PackageVersion: 9.0.89
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/a/Apache/Tomcat9/9.0.89/Apache.Tomcat9.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Oracle.JavaRuntimeEnvironment
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
With the dependency installed, the package installs and runs normally:
image

Verify: 
image

**Comment by jaov** [comment]
> Manual Validation ended with the same failure code `2`. Running the installer manually suggests that `Oracle.JavaRuntimeEnvironment` or another `JRE` might be a dependency: image

Should it also have an install dependency like Apache Netbeans does of a specific JRE?

**Pull request closed without merging**